fastlane/Fastfile

```diff
@@ -20,7 +20,7 @@ platform :ios do
       skip_build: true,
       output_directory: './artifacts/unit-tests',
       scheme: ENV['REM_FL_TESTS_SCHEME'] || 'Tests',
-      device: ENV['REM_FL_TESTS_DEVICE'] || 'REM SDK Unit Tests',
+      device: ENV['REM_FL_TESTS_DEVICE'] || 'iPhone 11',
```

Comment:
I know in our case this will work but since it's public repo maybe it's better to use `iPhone 8` for example to keep support for Xcode 10

Reply:
Sure. I'll do that.